py-polars/polars/expr/expr.py

```diff
@@ -4607,6 +4607,17 @@ def map_elements(
               `pl.col("col_name").sqrt()`.
             - For mapping inner elements of lists, consider:
               `pl.col("col_name").list.eval(pl.element().sqrt())`.
+            - For mapping elements of struct fields, consider:
+              `pl.col("col_name").struct.field("field_name").sqrt()`.
+
+            If you want to replace the original column or field,
+            consider `.with_columns` and `.with_fields`.
```

Comment:
could these be links? something like
```
:meth:`DataFrame.with_columns`
```
and something similar for `with_fields`

I can't remember the exact syntax, you may need to build the documentation locally to check it works